nj2k/testData/newJ2k/issues/kt-836.kt

```diff
@@ -4,7 +4,7 @@ import java.io.Serializable
 
 class Language(protected var code: String) : Serializable {
     override fun toString(): String {
-        return code
+        return this.code
```

Comment:
This change looks suspicious: an explicit `this` shouldn't be needed here. Please report it to YT.

Reply:
Issue created: https://youtrack.jetbrains.com/issue/KT-44170